libminifi/src/io/FileStream.cpp

```diff
@@ -74,6 +74,7 @@ FileStream::FileStream(std::filesystem::path path, uint32_t offset, bool write_e
   } else {
     file_stream_->open(path_, std::fstream::in | std::fstream::binary);
   }
+  length_ = 0;
```

Comment:
Why not initialize this on the initializer list?

Reply:
moved